src/Styles/Style.ts

```diff
@@ -0,0 +1,11 @@
+export const header1 = "text-2xl font-medium mb-1";
+
+export const primaryBtn = "my-2 m-auto block rounded-lg border border-transparent shadow-sm px-4 py-2 bg-blue-600 text-base font-medium text-white hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 disabled:opacity-50 w-10/12 sm:text-sm md:w-3/5 md:my-3 lg:w-1/3";
+
+export const secondaryBtn = "my-2 m-auto block rounded-lg border-2 shadow-sm px-4 py-2  border-blue-900 text-blue-900 font-medium hover:bg-blue-100 bg-blue-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 disabled:opacity-50 w-10/12 sm:text-sm md:w-3/5 md:my-3 lg:w-1/3";
+
+export const optionBtn = "w-full my-2 m-auto block rounded-lg border-2 shadow-sm px-4 py-2  border-blue-900 text-blue-900 font-medium hover:bg-blue-100 bg-blue-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 disabled:opacity-50 w-10/12 sm:text-sm md:w-3/5 md:my-3 lg:w-1/3"
+
+export const optionRight = "w-full my-2 m-auto block rounded-lg border-2 shadow-sm px-4 py-2  border-green-900 text-green-900 font-medium hover:bg-green-100 bg-green-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-green-500 w-10/12 sm:text-sm md:w-3/5 md:my-3 lg:w-1/3 cursor-not-allowed"
+
+export const optionWrong = "w-full my-2 m-auto block rounded-lg border-2 shadow-sm px-4 py-2  border-red-700 text-red-700 font-medium hover:bg-red-100 bg-red-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500 w-10/12 sm:text-sm md:w-3/5 md:my-3 lg:w-1/3 cursor-not-allowed"
```

Comment:
I really liked this and would love to implement this in my codebase as well as I am also using tailwind